test/Erc20Quest.spec.ts

```diff
@@ -18,11 +19,13 @@ describe('Erc20Quest', async () => {
   let deployedRabbitholeReceiptContract: RabbitHoleReceipt
   let expiryDate: number, startDate: number
   const mockAddress = '0x0000000000000000000000000000000000000000'
+  const mnemonic = 'announce room limb pattern dry unit scale effort smooth jazz weasel alcohol'
   const questId = 'asdf'
   const allowList = 'ipfs://someCidToAnArrayOfAddresses'
-  const totalRewards = 1000
-  const totalRewardsPlusFee = 1200
-  const rewardAmount = 10
+  const totalParticipants = 300
+  const totalRewardsPlusFee = 120000
```

Comment:
Shouldn't `totalRewardsPlusFee` be

```
totalParticipants * rewardAmount + (totalParticipants * rewardAmount * questFee / 10_000)
```
which in this test case would be
```
300 * 1000 + (300 * 1000  * 2000 / 10000)
```

Reply:
Yes it should, thanks for pointing this out. I updated the tests to use this math.